gcp/observability/opencensus.go

```diff
@@ -104,7 +104,7 @@ func startOpenCensus(config *config) error {
 	}
 
 	if config.CloudMonitoring != nil {
-		if err := view.Register(opencensus.ClientStartedRPCsView, opencensus.ClientCompletedRPCsView, opencensus.ClientRoundtripLatencyView); err != nil {
+		if err := view.Register(opencensus.ClientAPILatencyView, opencensus.ClientStartedRPCsView, opencensus.ClientCompletedRPCsView, opencensus.ClientRoundtripLatencyView); err != nil {
```

Comment:
Optional: You could define a slice with the views and pass it to register. That would be more readable. Not so imminent right now, but once you add a couple of more views, it will be.

```golang
views := []view.View{
  opencensus.ClientAPILatencyView,
  opencensus.ClientStartedRPCsView,
  opencensus.ClientCompletedRPCsView,
  opencensus.ClientRoundtripLatencyView,
}
if err := view.Register(views); err != nil { ... }
```

Reply:
Thanks for the suggestion. Will do this in the future. Especially once we add graceful shutdown (not officially speced but easy enough to add) - these will also get unregistered on shutdown to guarantee these recorded views send all their recorded data to exporter (which we flush to I think guarantee it blocks until all data is to backend), so will need to pull this to top level variable.

Reply:
See https://github.com/grpc/grpc-go/pull/6105/files#diff-fdee48b4c948042436440c4e033c7633fbffd1bae77521cedbb6136d27a9cd3eR38 for my incorporation for your suggestion